lib/fs.js

```diff
@@ -6,6 +6,7 @@
 const constants = process.binding('constants').fs;
 const util = require('util');
 const pathModule = require('path');
+const internalFs = require('internal/fs');
```

Comment:
I'm not sure I follow on the problem. As for a precedent, I'm sure it goes both ways, but as an example - https://github.com/nodejs/node/blob/a6d53c67795bf957b8b57d96814520f98069e80e/lib/child_process.js#L12-L17